test/AutoRest.TestServer.Tests/Mgmt/TestProjects/MgmtResourceGroupExtensionTests.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using AutoRest.CSharp.Input;
+using NUnit.Framework;
+using MgmtResourceGroupExtension;
+
+namespace AutoRest.TestServer.Tests.Mgmt.TestProjects
+{
+    public class MgmtResourceGroupExtensionTests : TestProjectTests
+    {
+        public MgmtResourceGroupExtensionTests() : base("MgmtResourceGroupExtension") { }
+
+        [Test]
+        public void TestResourceGroupExtensionTests()
```

Comment:
move this into TestProjectTests and make the validation generic

Reply:
Done